src/sentry/snuba/metrics/span_attribute_extraction.py

```diff
@@ -11,15 +11,15 @@
 # Matches the top level span attributes defined in Relay
 # https://github.com/getsentry/relay/blob/e59f21d9/relay-event-schema/src/protocol/span.rs#L119
 _TOP_LEVEL_SPAN_ATTRIBUTES = {
-    "span.exclusive_time",
-    "span.description",
-    "span.op",
-    "span.span_id",
-    "span.parent_span_id",
-    "span.trace_id",
-    "span.status",
-    "span.origin",
-    "span.duration",
+    "span.self_time": "span.exclusive_time",
```

Comment:
nit: This is now missing span.exclusive time mapped to itself, is that on purpose?

Reply:
yes, I assume that only `span.self_time` should be public